### DISCUSSION TITLE
Not able to install data-hub-export-bundle in pimcore

### DISCUSSION THREAD

**Discussion by gunnu1303** [pimcore/pimcore#14123] [Q&A / Support]
I'm trying to install data-hub-export-bundle but it's showing this error :-

Could not find a matching version of package pimcore/data-hub-file-export. Check the package spelling, your version constraint and that the package is available in a stability which ma tches your minimum-stability (stable).

Process Manager and Data Hub are both enabled and installed as per the [Pimcore documentation](https://pimcore.com/docs/data-hub-file-export/current/Installation/index.html)

composer.json

{
    "type": "project",
    "license": "GPL-3.0-or-later",
    "config": {
        "optimize-autoloader": true,
        "sort-packages": true,
        "process-timeout": 0
    },
    "require": {
        "basilicom/pimcore-data-quality-bundle": "^2.2",
        "ci-hub/simple-rest-adapter-bundle": "^2.0",
        "elements/process-manager-bundle": "^4.0",
        "pimcore/customer-management-framework-bundle": "^3.3",
        "pimcore/data-hub": "^1.5",
        "pimcore/data-importer": "^1.7",
        "pimcore/pimcore": "^10.0"
    },
    "require-dev": {
        "codeception/codeception": "^4.1.12"
    },
    "suggest": {
        "pimcore/data-hub": "Universal data interface for GraphQL, CSV and other formats"
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/",
            "Pimcore\\Model\\DataObject\\": "var/classes/DataObject"
        }
    },
    "scripts": {
        "post-create-project-cmd": "Pimcore\\Composer::postCreateProject",
        "post-install-cmd": [
            "Pimcore\\Composer::postInstall",
            "@pimcore-scripts"
        ],
        "post-update-cmd": [
            "Pimcore\\Composer::postUpdate",
            "@pimcore-scripts",
            "Pimcore\\Composer::executeMigrationsUp",
            "@pimcore-scripts"
        ],
        "pimcore-scripts": [
            "Pimcore\\Composer::clearCache",
            "Pimcore\\Composer::installAssets"
        ],
        "minimum-stability": "dev",
        "prefer-stable": true
    }
}

**Comment by frithjof** [comment]
i think this this an enterprise bundle

**Reply by NiklasBr** [reply]
Installation instructions are a bit unclear, we resolved it by first explicitly requiring `pimcore/enterprise-subscription-tools` in Composer, then configuring the enterprise key, and last installing the enterprise bundles like the data-hub-export-bundle.

**Reply by gunnu1303** [reply]
Hey Thankyou for the reply. But I'm curious to know that I have installed pimcore. Pimcore data-hub bundle and data-import bundle was not in built. So, I installed them by running composer require pimcore/data-importer like that. So, why I'm facing difficulty in installing data export bundle. Data-import was also not included in community edition as like data-export bundle. Please answer my question about it.

**Reply by NiklasBr** [reply]
I don't think I fully understand what your question is. The Datahub File Export bundle requires Pimcore Enterprise (a paid version of Pimcore). Data Import does *not* require Enterprise. Are you sure you are not getting them mixed up? Do you have the Enterprise version?

**Reply by gunnu1303** [reply]
Thank you for your response, sir. You have cleared up my confusion. I had no idea that data-importer and data-hub are part of the community edition, whereas data-export is part of the enterprise edition.
